SerialPrograms/Source/PokemonSV/Programs/TeraRaids/PokemonSV_TeraRoutines.cpp

```diff
@@ -651,6 +651,35 @@ TeraResult run_tera_summary(
 }
 
 
+void run_from_tera_battle(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context){

```

Comment:
This requires having your cursor over the run option right? Based on the naming of this function, it's probably better to move the logic to navigate to the start point into here.

Reply:
Shifted menu detection and cursor movement under this function as well